.git-blame-ignore-revs

```diff
@@ -1,2 +1,4 @@
 # ocamlformat 0.25.1
 3f01f6f3694e48cb1868018121f3959f8d23baca
+# Upgrade to OCamlformat 0.26.0
+8a1a919e3895787ae3781da1d6aff4cc41ce04c3
```

Comment:
No need to do that in this PR since this will change

Reply:
Right!

I'll make sure to remove that and fix the version line before I mark the PR as ready.

Reply:
Though, do you think this PR could be merged unsquashed ?

Reply:
for now I prefer doing this in 2 steps

Reply:
Ok!